README.md

```diff
@@ -58,111 +57,11 @@ import * as PXBThemes from '@pxblue/react-native-themes';
-...
-<Badge size={24} visible theme={theme.dark ? blueDarkAlt : {}}></Badge>
-```
-
-##### Component-Based Usage
-
-If you are using components frequently, it's best to create a wrapper component that will handle the alternate theme logic. This will allow you to keep your code more readable and avoid errors with inconsistent theme application.
-
-To do this, you define a wrapper component that acts as a pass-through for all of the default props and adds the theme logic.
-
-```tsx
-import React from 'react';
-import { blueDarkAlt } from '@pxblue/react-native-themes';
-import { SomeComponent as PaperComponent, useTheme } from 'react-native-paper';
-...
-export const SomeComponent: typeof PaperComponent = (props) => {
-    const theme = useTheme(props.theme);
-    return (
-        <PaperComponent
-            {...props}
-            theme={
-                theme.dark
-                ? Object.assign({}, blueDarkAlt, props.theme)
-                : {}
-            }
-        />
-    );
-};
-```
-
-You would then use your custom wrapper component throughout your application instead of using the component directly from React Native Paper:
-
-```tsx
-import { SomeComponent } from './path/to/SomeComponent';
-...
-<SomeComponent {...samePropsAsThePaperComponent} />
-```
-
-The `Button` component is a special case that requires the alternate theme only if you are using the contained mode. The wrapper component for the `Button` should look like:
-
-```tsx
-import React from 'react';
-import { blueDarkAlt } from '@pxblue/react-native-themes';
-import { Button, useTheme } from 'react-native-paper';
-...
-export const MyCustomButton: typeof Button = (props) => {
-    const theme = useTheme(props.theme);
-    return (
-        <Button
-            {...props}
-            theme={
-                props.mode === 'contained' && theme.dark
-                    ? Object.assign({}, blueDarkAlt, props.theme)
-                    : {}
-            }
-        />
-    );
-};
-```
-
-The `TextInput` component is a special case that requires usage of both `blueDark` and `blueDarkAlt` themes. The wrapper component for the `TextInput` should look like:
-
-```tsx
-import { blueDark, blueDarkAlt } from '@pxblue/react-native-themes';
-import { TextInput, useTheme } from 'react-native-paper';
-import _clonedeep from 'lodash.clonedeep';
-
-export const MyCustomTextInput: typeof TextInput = (props) => {
-    const theme = useTheme(props.theme);
-    const darkTheme = _clonedeep(blueDarkAlt);
-    darkTheme.colors.primary = blueDark.colors.primary;
+### React Native Paper Wrapper Components
 
-    return <TextInput {...props} theme={theme.dark ? Object.assign({}, darkTheme, props.theme) : {}} />;
-};
-```
+The theming mechanism for React Native Paper is a bit limited in the amount of customization available for components. Because of this, we updated the `ThemeColors` interface and created wrapper components to use in place of the standard React Native Paper components.
```

Comment:
```suggestion
The default theme structure for React Native Paper components does not offer us enough control to make some components look exactly the way they should for PX Blue applications. Because of this, we have extended the default theme type definition (see below) and created wrapper components with the correct styles to use in place of the standard React Native Paper components.
```